assets/svelte/wal_pipelines/Form.svelte

```diff
@@ -105,9 +103,9 @@
       </CardHeader>
       <CardContent>
         <p class="mb-4 text-secondary-foreground text-sm" class:hidden={isEdit}>
-          With a Change Capture Pipeline, you can capture every insert, update,
-          or delete that happens to one or more tables into another table in
-          your database. Then, you can stream these events with Sequin.
+          With Change Retention, you can capture every insert, update, or delete
+          that happens to one or more tables into another table in your database
+          so you can backfill and replay changes to a sink.
```

Comment:
```suggestion
          With Change Retention, you can capture every insert, update, or delete
          from one or more tables into another table in your database.
          Then, you can backfill and replay changes to a sink.
```

I like the two shorter sentences